cf-oscore/src/main/java/org/eclipse/californium/oscore/ContextRederivation.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * Copyright (c) 2018 RISE SICS and others.
```

Comment:
Switch to 2019?

Reply:
Done in commit 2cb4bc043454e7302e201bc86f6ca4af19a24fe0